contracts/periphery/contracts/s2s/SmartChainXApp.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0;
+
+import "./SmartChainXLib.sol";
+
+abstract contract SmartChainXApp {
+    struct Config {
+        // config for dispatch call
+        address dispatchAddress;
+        bytes2 dispatchCallIndex;
+        // config for market fee
+        address storageAddress;
```

Comment:
Should be storage precompile?

Better documentation and naming.

Reply:
market fee is from storage precompile

Reply:
Yes, but the meaning of this settings is `state_storage` precompile, the name and doc is miss leading IMO.

Reply:
yes, I will update the inline docs for the whole sdk.